app/views/layouts/application.haml

```diff
@@ -87,7 +87,9 @@
                   %li
                     = link_to notifications_path do
                       Notifications
-                      - unread_notif_count = current_user.notifications.unread.visible_to(current_user).count
+                      - unread_notifs = current_user.notifications.unread.visible_to(current_user)
+                      - unread_notifs = unread_notifs.not_ignored_by(current_user) if current_user&.hide_from_all
```

Comment:
I don't think we need the `&.` here since it's within a `logged_in?` block

Reply:
Oh probably not I just copied it over from notifications#index